app/views/messages/_message.html.erb

```diff
@@ -13,11 +13,31 @@ end %>
 <% scroll_down = scroll_down || initial_page_load %>
 <% thinking ||= last_message && message.content_text == "" && message.not_cancelled? %>
 <% done_thinking = !thinking && message.content_text.present? %>
-<% debug_tool_messages = false %>
```

Comment:
Let's keep this debug in place, I've found it quite helpful when working on new tools